pygama/dsp/_processors/get_multi_local_extrema.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from numba import guvectorize
+from pygama.dsp.errors import DSPFatal
+
+
+
+@guvectorize(["void(float32[:], float32, float32[:], float32[:], float32[:], float32[:], float32[:], float32[:])",
+              "void(float64[:], float64, float64[:], float64[:], float64[:],float64[:], float64[:], float32[:])"],
+             "(n),(),(m) -> (m),(m),(),(),()", nopython=True, cache=True)
+def get_multi_local_extrema(w_in, delta_in, t_arbitrary_in, vt_max_out, vt_min_out, n_max_out, n_min_out, flag_out):
```

Comment:
is delta an amplitude? If so I would use a_delta_in

We need to define a convention for a variable such as flag_out. @iguinn @ggmarshall, any suggestion?